action.yml

```diff
@@ -0,0 +1,14 @@
+name: 'Commit and push to branch'
+description: 'Commits and pushes the result of the action to a remote branch.'
+author: '@automattic/jetpack'
+inputs:
+  branch:
+    description: 'Branch ref to push the changes to'
+    required: true
```

Comment:
Should there be a default value here?

Reply:
The only logistical default I can think of is the current branch it's standing on, but I'd be afraid that it could ruin's someone's day if they are not expecting it? 

Do you have an idea what would make sense as a default?

Reply:
We decided to leave this as required with no default.